cardea-data/src/main/java/ca/on/oicr/gsi/cardea/data/Case.java

```diff
@@ -39,18 +39,23 @@ private Case(Builder builder) {
     this.id = requireNonNull(builder.id);
     this.donor = requireNonNull(builder.donor);
     this.projects = unmodifiableSet(builder.projects);
-    this.assay = requireNonNull(builder.assay);
-    this.assayId = this.assay.getId();
+    this.assay = builder.assay;
+    if (this.assay != null) { // assay is left null within Cardea to reduce output size of data
+      this.assayId = this.assay.getId();
+    } else {
+      this.assayId = builder.assayId;
+    }
```

Comment:
This will result in `assay` always being null in Dimsum, which will break several things in Dimsum (check references to `Sample#getAssay` in current `main`). It either needs to get populated on the Dimsum side, or the `assay` field and `getAssay` should be removed, and anywhere it was used in both Dimsum and Cardea needs to be updated to get the assay in a different way. I think we should go with the latter option so we don't need to rebuild all of the cases on the Dimsum side